containers/dotnet-fsharp/README.md

```diff
@@ -9,7 +9,7 @@
 | *Contributors* | The VS Code Team, F# team |
 | *Categories* | Languages |
 | *Definition type* | Dockerfile |
-| *Published image architecture(s)* | x86-64 |
+| *Published image architecture(s)* | x86-64, arm64/aarch64 |
```

Comment:
We'll need to qualify which images have arm64/aarch64 support since the focal ones will not.  See the python-3 definition for how this is described.